docs/samples/Microsoft.ML.Samples/Dynamic/ProjectionTransforms.cs

```diff
@@ -37,7 +37,7 @@ public static void Example()
             };
 
             // A pipeline to project Features column into Random fourier space.
-            var rffPipeline = ml.Transforms.Projection.CreateRandomFourierFeatures(nameof(SamplesUtils.DatasetUtils.SampleVectorOfNumbersData.Features), newDim: 4);
+            var rffPipeline = ml.Transforms.RandomFourierExpand(nameof(SamplesUtils.DatasetUtils.SampleVectorOfNumbersData.Features), dimension: 4);
```

Comment:
>dimension [](start = 134, length = 9)

if it's `rank` in Whitening and PCA, maybe it also should be `rank`?

Reply:
Or `dimension` in Whitening and PCA

---
In reply to: [263557554](https://github.com/dotnet/machinelearning/pull/2865#discussion_r263557554) [](ancestors = 263557554)

Reply:
Let's do rank.